aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
+
+
+class DAPCA4Cost(Model):
+    """
+    Cost Model from Raymer, Aircraft Design: A Conceptual Approach 3rd Edition,
+    pp 713-714
+
+    Note: this is also one instance of the RAND Corp cost models.
+
+    The total cost is Research, Development, Test and Evaluation Cost
+
+    Cost is in 2012 dollars
+
+    Engineering hours considers all airframe, propulsion and avionics
+    integration and design.  It does not include engineering of avionics,
+    propulsion, tooling or production planning.
+
+    Tooling hours covers all the preparation for production.  It also covers
+    ongoing tooling cost through production.
+
+    Manufacturing hours is the direct labor to fabricate the aircraft. It also
+    includes cost of labor of subcontractors.
+
+    Quality Control hours covers all inspections.
+
+    Flight test cost covers all cost to prove airworthiness.
+
+    Manufacturing Materials is the raw materials cost and assumes that airframe
+    and main materials are made of aluminum.
+
+    The Engine Cost assumes that the cost of the engine is not known.
+
+    Uses 2012 pricing and economics
+
+    Units are not included in variable definitions
```

Comment:
I think I added this line before we decided to `disable_units()`.  What I meant to say is that because `$` is not defined, variables that are in units of `$` are not defined with units of `$`.